src/DateUtils.js

```diff
@@ -92,13 +92,13 @@ module.exports = {
             monthName: months[date.getMonth()],
             day: date.getDate(),
             fullYear: date.getFullYear(),
-            time: showTwelveHour ? twelveHourTime(date) : this.formatTime(date),
+            time: this.formatTime(date, showTwelveHour),
         });
     },
 
     formatTime: function(date, showTwelveHour=false) {
         if (showTwelveHour) {
-          return twelveHourTime(date);
+            return twelveHourTime(date);
```

Comment:
spurious spaces?

Reply:
it was indented with 2 spaces, wrongly, so I fixed to 4

Reply:
oh, right you are